grouper/repositories/service_account.py

```diff
@@ -84,6 +84,27 @@ def mark_disabled_user_as_service_account(self, name, description="", mdbset="")
 
         user.is_service_account = True
 
+    def owner_of_service_account(self, name):
+        # type: (str) -> str
+        owner = (
+            self.session.query(Group.groupname)
+            .filter(
+                SQLServiceAccount.user_id == SQLUser.id,
+                SQLUser.name == name,
+                GroupServiceAccount.service_account_id == SQLServiceAccount.id,
+                Group.id == GroupServiceAccount.group_id,
+            )
+            .scalar()
+        )
+        if not owner:
+            raise ServiceAccountNotFoundException(name)
+        return owner
+
     def service_account_exists(self, name):
         # type: (str) -> bool
         return SQLServiceAccount.get(self.session, name=name) is not None
+
+    def service_account_is_enabled(self, name):
+        # type: (str) -> bool
+        user = SQLUser.get(self.session, name=name)
+        return bool(user and user.is_service_account and user.enabled)
```

Comment:
Huh, out of curiosity why is the `bool()` necessary? Quick poking and Python3 seems to understand that `obj and bool and ...` is `bool`